src/components/app/DetailsContainer.css

```diff
@@ -0,0 +1,27 @@
+.DetailsContainer .layout-pane > * {
+  box-sizing: border-box;
+  width: 100%;
+  height: 100%;
+}
+
+.DetailsContainer .layout-pane:not(.layout-pane-primary) {
+  max-width: 250px;
+}
+
+/* overriding defaults from splitter-layout */
+.DetailsContainer {
+  /* SplitterLayout injects position: absolute, this conflicts with our using
+   * Flex Layout. */
+  position: unset;
+}
+
+.DetailsContainer .layout-splitter {
+  background: var(--grey-10); /* Same background as sidebars */
+  border-left: 1px solid var(--grey-40);
```

Comment:
Ah no, sorry, I've indeed put some thought for this before and for this PR I merely copied it from what I did before. I'll try what you suggest nevertheless, but in a separate PR so that we don't change too much things in this PR. Is that good ?

Reply:
So, I tried it. The border-top is definitely good and we should have it.

But I'm not so sure about the color change; I thought that a darker grey would set the sidebar slightly apart, and in the same time bring an affordance that it's resizable.
What do you think ?

Reply:
> So, I tried it. The border-top is definitely good and we should have it.

🎉 

> But I'm not so sure about the color change; I thought that a darker grey would set the sidebar slightly apart, and in the same time bring an affordance that it's resizable.
What do you think ?

Lets try with your suggestion and we can still change it to a lighter color, when it feels too heavy. :)

> I'll try what you suggest nevertheless, but in a separate PR so that we don't change too much things in this PR.

I know, we want to ship as soon as possible, but as this and the one above seem like just small changes, I think it would be good to ship it with it, so we can deploy soon and have the best user experience possible without much additional effort?